### ISSUE TITLE
Cannot set ImageGlass as default photo viewer if Photos is the current default app

### ISSUE THREAD

**Issue by jungheeMW** [d2phap/ImageGlass#1912]
### System information
- Windows OS version: Windows 10 Home OS build 19045.4291
- ImageGlass version: 9.0.11.502 (64-bit)
- ImageGlass release: Classic

### To Reproduce
1. Make sure the current default photo viewer is "Photos" (This is the Windows 10 default photo viewer)
2. (Optional: Run ImageGlass as administrator)
3. Settings > File type associations > Default photo viewer > Make default

### Actual behavior
1. If Photos is the current default photo viewer, attempting to set ImageGlass to the default photo viewer will show it as successful, but does not alter the default photo viewer.
2. Instead, it adds ImageGlass to the list of selections of default apps. The user still has to manually change their default photo viewer in the Default apps section of the Settings panel.
3. If the default photo viewer is empty, or set to a different app, using ImageGlass to set it as the default photo viewer will be successful.

### Expected behavior
Setting ImageGlass as the default photo viewer will actually set it as the default photo viewer, without having to go into the Settings panel.

### Screenshots / Video / Sample image file
<img width="673" alt="ig bug" src="https://github.com/d2phap/ImageGlass/assets/134947146/adf17bbc-fdd6-49ae-9957-0ae112fb07d6">
<img width="960" alt="results1" src="https://github.com/d2phap/ImageGlass/assets/134947146/56d36962-cc20-4e28-8122-9d06f489eafb">

**Comment by d2phap** [comment]
Since Windows 10, MS has made it more difficult to change default app. It prevents the third-party app changing default app automatically.

So the "Set as default photo viewer" just register ImageGlass to the system, its supported formats to be listed in the Default apps > Photo viewer dropdown

**Comment by jungheeMW** [comment]
Could the wording be changed then? Instead of "Set as default photo viewer", maybe reword it to "Add as a default photo viewer" or "Register as a default photo viewer"?

Or a notification/message of some kind that lets the user know they still have to manually change it in the App settings.
